### PR TITLE
Added detail on problem types/file requirements/output to README

### PR DESCRIPTION
**Pull request by MattRolchigo**
* Updated README.md to include more details on ExaCA output data 
* Updated and examples/README.md to include more details on specific problem types (C, S, and R), along with requirements for ExaCA temperature, orientation, and material data files

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+* Problem type S is an array of hemispherical spots, with the number of spots in X, Y, and the number of layers for which the pattern is repeated (offset by a specified number of cells in the positive Z direction) specified. This problem type also uses fixed thermal gradient magnitude and cooling rate for each spot. 
+* Problem type R is a custom solidification problem using time-temperature history file(s) (default location is `examples/Temperatures`). The time-temperature history can consist of one file, repeated "Number of Layers" times, with each additional layer offset by "Layer height" cells in the +Z direction, or a pattern of temperature files. For example, if a simulation with alternating even and odd layer time-temperature histories is desired, the file names should be prepended with "1" and "2", and the "Number of temperature files in series" line in the input file should be set to 2. The temperature files should be of the following form:
+    ** The first line should be the names of the columns: x, y, z, tm, tl, cr
```
Comment (streeve) [inline examples/README.md:7]: This isn't formatting correctly - should still be one *

**Review by streeve — CHANGES_REQUESTED** [review]
Can you add the required column to each problem type? Then let's merge this first and rebase the temperature init

**Comment by MattRolchigo** [comment]
Outdated with recent/pending changes

**Comment by streeve** [comment]
Would still prefer merging this separate from the temperature updates

**Review by streeve — APPROVED** [review]

**Pull request merged**